test/TestServerProjectsLowLevel/paging/Generated/PagingClient.cs

```diff
@@ -1077,26 +1077,26 @@ IEnumerable<Page<BinaryData>> CreateEnumerable(string nextLink, int? pageSizeHin
         /// 
         /// </remarks>
 #pragma warning disable AZC0002
-        public virtual AsyncPageable<BinaryData> GetMultiplePagesFragmentNextLinkAsync(string apiVersion, string tenant, RequestContext context = null)
+        public virtual AsyncPageable<BinaryData> GetMultiplePagesFragmentNextLinkAsync(string tenant, string apiVersion, RequestContext context = null)
```

Comment:
This feels dangerous to me - existing code will compile but pass the wrong values for `tenant` and `apiVersion`.  Which of our new rules is dicating this?

Reply:
`tenant` is defined in path, `apiVersion` is defined in query.

Reply:
What is our plan to make sure all the relevant code is updated correctly in our generated libraries, since we won't have compiler support for this?

Reply:
I see.  So this shouldn't have any effect on existing public APIs?  If not, then I'm not concerned.